.travis.yml

```diff
@@ -31,16 +26,16 @@ cache:
         - $HOME/.composer/cache
 
 env:
-    - GUSH_HOME=/tmp/gush GUSH_CACHE_DIR=/tmp/gush/cache
+    - GUSH_USE_FS=true
+    - GUSH_USE_FS=false
 
 before_install:
     # this is needed because something is broken in hhvm
-    #- composer self-update
+    - export TMPDIR=/tmp # Will this make the tests work please?
```

Comment:
Oops forgot to remove that one :) It actually turned out to be a wrong usage of the tests.
So this can be removed ;)